src/Microsoft.Diagnostics.Monitoring.Options/GlobalCounterOptions.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Text;
+
+namespace Microsoft.Diagnostics.Monitoring.WebApi
+{
+    public class GlobalCounterOptions
```

Comment:
This needs to be added to https://github.com/dotnet/dotnet-monitor/blob/main/src/Tools/dotnet-monitor/ConfigurationJsonWriter.cs